entries/s/standardnotes.com.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Standard Notes": {
+        "domain": "standardnotes.com",
+        "tfa": [
+            "totp"
+        ],
+        "documentation": "https://standardnotes.com/help/25",
```

Comment:
```suggestion
        "documentation": "https://standardnotes.com/help/25",
        "recovery": "https://standardnotes.com/help/22",
```

Reply:
It says these changes are invalid because no change was made to the code

Reply:
I just went ahead and added this myself